output/plugins/forms/xml/forms.cppcode

```diff
@@ -407,7 +407,26 @@ Written by
 			#ifequal $aui_managed "1"
 			@{ @#include &lt;wx/aui/aui.h&gt; @}
 		</template>
-		<template name="settings">
+        <template name="code_init">
+            #ifnotnull $create_children
```

Comment:
This test is useless, a boolean property can never be null

Reply:
What if it doesn't exist as in when this change is first implemented?

Reply:
The properties are part of the object model that gets defined in the XML files. If the XML file defines the property, it is available with a valid value, no matter if your project file contains a value for it or not. If your code template tries to access a property that is not defined in the XML files, this is a programmer error and results into strange errors. These errors can't be prevented by such a check because the check itself already causes an error.